### PR TITLE
Redux in rect components

### PR DESCRIPTION
**Pull request by Said-laasri**
**REDUX IN REACT COMPONENT:** 

-  keep state immutable in reducers!
- Import Redux Provider and the store in the main component of your React app.
- Connect the app to the Redux store.
- list of books displays books from the Redux store.
- Use useSelector to consume the state.
- Import Redux useDispatch.
- Set the data inputs in the local React state (set title and author).
- In your React component responsible for removing books, add functionality for the "Remove" button to click
- In your React component responsible for categories, add functionality for the "Check status" button click:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ to run in terminal / git :
```
Comment (julie-ify) [inline README.md:43]: - [x] Kindly edit your readme documentation to match this project. Your Readme has details about math-magicians and Tv-show project from JS. Kindly update with proper details related to this project.

- [x] Remember to include instructions on how to clone the app and run the app locally.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Said-laasri,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Said-laasri) [reply README.md:43]: @julie-ify  thanks for reviewing.
my bad for not paying attention to my readme file, I fix it thanks again

**Review by elmar8287 — APPROVED** [review]
Hi, @Said-laasri.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- You implemented all suggestions from the previous review 👍 

### Optional suggestions

- Please, consider fixing the broken link to the MIT.md file 👍 
Screen Shot 2022-08-16 at 13 20 45



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Said-laasri** [comment]
@elmar8287  thanks for the review I did change the link to MIT license

**Comment by Said-laasri** [comment]
@elmar8287  Thank you for reviewing my project.

**Pull request merged**